Nulecule.yaml

```diff
@@ -0,0 +1,95 @@
+# Put all metadata here (no more separate id, specversion objects, etc.
+metadata:
+  name: Hello World
+  id: helloworld
+  appversion: 0.0.1
+  spec: 1.0.0
+  description: Application
+
+# All provider details / what provider to orchestrate to. Because let's face it, 99% of the time the user is only focused on one specified provider when deploying Atomic App
+provider:
+  default: kubernetes
+
+  # This will be encrypted on build / won't be shown to any user that uses it. This is also OPTIONAL. With providing "secrets" this makes applications super portable. Got to find someone whom is good at encryption to implement this.
+  secrets: 
+    provider-api: https://localhost:8080
+    provider-auth: foobar
+    provider-tlsverify: true
+    provider-cacert: |
+      ----BEGIN CERTIFICATE REQUEST-----
+      FOO
+      -----END CERTIFICATE REQUEST-----
+
+# Global parameters (hi rtnpro!)
+params: 
+  db_user:
+    description: Database username
+  db_pass:
+    description: Database password
+  db_name:
+    description: Database name
+  db_host:
+    description: Database service name
+    default: mariadb
+ 
+# Inheriting a container that's already been "nuleculized"
+db:
+  source: docker://projectatomic/mariadb:latest
+  # Other possible providers:
+  # source: rkt://projectatomic/mariadb:latest
+  # source: file://foo/bar/container
+  # source: https://localhost/container.tar.gz
+
+# MANUALLY: Creating a Nulecule container with artifact files (variable substition which is what we use at the moment.
+etherpad:
+  build: manual
+
+  params: 
+    image:
+      description: Container image
+      default: centos/etherpad
```

Comment:
Probably shooting in the dark but, instead of 
~~~
params: 
     image:
       description: Container image
       default: centos/etherpad
~~~
is it possible to have something simple like 
~~~
image: centos/etherpad
~~~